examples/react-testing-library/.babelrc.js

```diff
@@ -1,6 +1,4 @@
-// Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
```

Comment:
don't remove the copyright notices

Reply:
Just want to make sure you saw the comment about `examples/` being excluded from the copyright checker.

If we want to keep them, should we use the latest template then?

```
/**
 * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
```

Reply:
Nah, just keep the as is 🙂

Reply:
fcd0e09. It should be like nothing happened now! 😄